test/unit/definitions/divorce/AuthorisationCaseType.test.js

```diff
@@ -1,7 +1,7 @@
 const expect = require('chai').expect;
 const { uniqWith } = require('lodash');
 
-const caseTypeTab = Object.assign(require('definitions/divorce/json/AuthorisationCaseType'), {});
+const caseTypeTab = Object.assign(require('definitions/divorce/json/AuthorisationCaseType/AuthorisationCaseType'), {});
```

Comment:
as above

Reply:
Hi @yarpo I've changed this slightly to pickup all the files in the corresponding folder - which it didn't do before. Also tweaked the test so you only have to change the one line (the first Object.assign) when swtiching between production and non-prod files in the test. Can you please let me know what you think, I've done the change for AuthorisationCaseType, AuthorisationCaseState and AuthorisationCaseField and I've tested in prod and non-prod layouts.